azure/kubernetes/aks-kubernetes.md

```diff
@@ -0,0 +1,211 @@
+| Azure Kubernetes Service RBAC Cluster Admin   |Allows super-user access to perform any action on any resource. It gives full control over every resource in the cluster and in all namespaces. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+||||
+
+
+Secondly, there is the Kubernetes RBAC system.
+This permits fine-grained control of the Kubernetes resources themselves via Role and ClusterRole definitions.
+These are then linked with users via integration with Active Directory by RoleBindings and ClusterRoleBindings.
+
+There is a feature in Preview which integrates the Kubernetes RBAC authorization with Azure RBAC [1].
+This means you can define Kubernetes roles in the same system you define roles for Azure resources, and AKS will use Azure to authorize Kubernetes access requests.
+
+1.  https://docs.microsoft.com/en-us/azure/aks/concepts-identity
+2.  https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#containers
+3.  https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+
+### Privileged Access Management
+
+As discussed, identities should be managed via Azure Active Directory.
+Azure provides guidance and AD functionality for managing privileged identities, such as the Privileged Access Management notification service.
+
+1.  https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-admin-roles-secure
+
+## Encryption & secure data management
+
+### Encryption of data at rest
+
+AKS allows connecting pods to persistent storage via volumes.
+Volumes are backed by the Azure Storage product, which encrypts all data as standard[4].
+The encryption, decryption, and key management processes are transparent to users.
+Customers can also choose to manage their own keys using Azure Key Vault.
+There are a variety of methods for authenticating with the Storage platform.
+
+Note that this also applies to fully managed components, such as etcd[3].
+
+1.  https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction
+2.  https://docs.microsoft.com/en-gb/azure/aks/concepts-storage
+3.  https://docs.microsoft.com/en-us/azure/aks/concepts-security#kubernetes-secrets
+4.  https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption
+
+### Encryption of data in transit
+
+In-transit encryption is generally more an application layer concern and not considered by the lower-layer routing and network services that Kubernetes offers.
+However, it does support "ingress controllers" which can process data at the application level, including decryption of TLS for routing to pods.
+This allows centralised management of TLS at the Kubernetes networking level, even for pod applications which do not support it.
+
+1.  https://docs.microsoft.com/en-gb/azure/aks/concepts-network
+
+### Certificate and key management
```

Comment:
Certificate and Key Management 

or

Certificate & Key Management

We need to keep it consistent across the document.

Reply:
@TLATER - I have addressed in the following PR to your fork ...

- https://github.com/TLATER/cloud-service-certification/pull/2